src/io/serialization/versioning.ts

```diff
@@ -21,6 +22,7 @@ export interface Serializers {
 export const SERIALIZERS = new Map<string, Serializers>(
   Object.entries({
     ...v2_0_0,
+    ...v2_1_0,
   }),
 )
```

Comment:
We maintain a list of all serializers, in order to route incoming objects depending on their versions. The serializer v2.0.0 was missing, so the I/O on the objects of v2.0.0 was always failing. This fixes that problem.